Examples/Multiplayer/CSharp/Csp-Unity-VisionPro-Example/Assets/Scripts/HelloWorld.cs

```diff
@@ -21,7 +21,7 @@ public class HelloWorld : MonoBehaviour
     [SerializeField] private LocalPlayer localPlayerPrefab;
     [SerializeField] private RemotePlayer remotePlayerPrefab;
     
-    [SerializeField] private TextMeshProUGUI titleTenentReplaceLabel;
+    [SerializeField] private TextMeshProUGUI[] titleTenentReplaceLabels;
```

Comment:
The sign in UI also had a location to substitute the Tenant ID.
![Screenshot 2024-06-07 at 2 38 09 PM](https://github.com/magnopus-opensource/connected-spaces-platform/assets/100873114/ee3e824c-07c8-4e83-bc58-8386f0170890)